README.md

```diff
@@ -179,14 +161,50 @@ Note that if you are running `extend_ca_cert` locally on the primary Puppet serv
 bolt plan run ca_extend::extend_ca_cert --targets local://hostname --run-as root
 ```
 
-### ca_extend::upload_ca_cert Plan
+### Distribute `ca.pem` to agents 
+
+Next, distribute `ca.pem` to agents using one of the three methods:
 
-You can use this plan with `cert` parameter to specify the location of the updated CA cert and distribute it to the nodes specified in the `targets` parameter
+#### 1. Using the ca_extend::upload_ca_cert Plan
+
+You can use this plan with `cert` parameter to specify the location of the updated CA cert and distribute it to the nodes specified in the `targets` parameter. The `ca_extend::upload_ca_cert` plan works best with a Bolt [inventory file](https://puppet.com/docs/bolt/latest/inventory_file.html) to specify targets; this allows for simultaneous uploads to \*nix and Windows agents. See the Bolt documentation for how to configure an inventory file. Alternatively, you may specify targets for the `ca_extend::upload_ca_cert` plan by connecting Bolt to [PuppetDB](https://puppet.com/docs/bolt/latest/bolt_connect_puppetdb.html), after which the [--query](https://puppet.com/docs/bolt/latest/bolt_command_reference.html#command-options) option can be used. Lastly, you may instead use an `ssh` config file if you will only use `ssh` transport to upload the CA certificate to agents. Bolt defaults to using the `ssh` transport, which in turn will use `~/.ssh/config` for options such as `username` and `private-key`.
 
 ```bash
 bolt plan run ca_extend::upload_ca_cert cert=<path_to_cert> --targets <TargetSpec>
 ```
 
+#### 2. Manually deleting `ca.pem` on agents and letting them download that file as part of the next Puppet agent run
+
+The agent will download `ca.pem` only if it is absent, so it must be deleted to use this method. 
+
+For example, on an \*nix agent node delete `ca.pem` by running:
+
+```bash
+rm -rf $(puppet config print localcacert)
```

Comment:
Could we remove the `-rf` options?  We shouldn't need recursive or force for a regular file.